### ISSUE TITLE
Incomplete build instructions.

### ISSUE THREAD

**Issue by inori-aizawa** [Feodor2/Mypal68#414]
Please complete the build instructions in the wiki. Thanks.

**Comment by Feodor2** [comment]
Where you are stuck?

**Comment by inori-aizawa** [comment]
![immagine](https://github.com/Feodor2/Mypal68/assets/155257081/942ae123-6f55-46d3-ba85-1a89211062f5)

**Comment by Feodor2** [comment]
Are you already have built the rust?

**Comment by inori-aizawa** [comment]
No, and I would also like to have more detailed instructions on how to build the Rust compiler, plus where you got the Rust 1.35 source code from.

**Comment by Feodor2** [comment]
Okey, then skip it, because that became outdated, i am to rewrite it all, so far take regular rust 1.45.
Are you really have other things ready, cbindegn for example?

**Comment by inori-aizawa** [comment]
Yes, I do.

**Comment by inori-aizawa** [comment]
How do I fix this?
![immagine](https://github.com/Feodor2/Mypal68/assets/155257081/1d2ee76e-654c-4742-ab17-7948bfc61b14)

**Comment by Feodor2** [comment]
Do you want to say that you already have done things showed there and it still not fixed?

**Comment by Feodor2** [comment]
@bluedragon-cairo 
>If you have Notepad++...

This is wrong, must do exactly as showed on screen (four git commands)

**Comment by K4sum1** [comment]
> so far take regular rust 1.45.

Can you give updated patches for 1.45 please?

**Comment by Feodor2** [comment]
[0001-rust145.patch](https://github.com/user-attachments/files/16180782/0001-rust145.patch)
[0001-rust1451.patch](https://github.com/user-attachments/files/16180785/0001-rust1451.patch)
It is a bug that sometimes rust fails to find the crete std, I did loging it (second patch to remove logging), but with it error does not manifest, no time to dig further, restart and all

**Comment by K4sum1** [comment]
Thanks, I was able to build 1.45.

I got it onto my build VM, however I am unable to get cbindgen to build. What version of cbindgen do you have, and if possible could I have your built copy pls?

I'm also not entirely sure what you mean compile without cargo. I'm pretty sure cargo is hard requirement but idk.

**Comment by Feodor2** [comment]
v20, cargo is the same as the 'make' with 'c', it compiles nothing, only working with makefiles

**Comment by Feodor2** [comment]
If you going to build for winxp, pay attention for llvm - version above 9 fails on winxp, something about "sesuretu" protection, again 9 still fine, so droped above.
